topics/asciidoc.adoc

```diff
@@ -1,15 +1,21 @@
-[[asciidoc]]
+[asciidoc]
```

Comment:
Does this syntax work? I thought it had to be:

[[anchor]]
[#anchor]
or
[id="anchor"]

Reply:
@ncbaratta I tested and while it works with AsciiDoctor, it breaks the TOC when built using CCUTIL.
`[#anchor]` works consistently with AsciiDoctor and CCUTIL.
I'll fix the anchor formatting.